packages/gatsby-transformer-sqip/package.json

```diff
@@ -24,7 +24,7 @@
     "debug": "^3.2.6"
   },
   "peerDependencies": {
-    "gatsby": "^2.0.0",
+    "gatsby": "^2.0.15",
     "gatsby-source-contentful": "^2.0.0",
```

Comment:
Let's add `gatsby-core-utils` as a dependency. You can do this by running `yarn add gatsby-core-utils` inside this folder.

Reply:
I keep getting this error `error https://github.com/okonet/listr-update-renderer/tarball/upgrade-log-update: Integrity check failed for "listr-update-renderer" (computed integrity doesn't match our records, got "sha1-Bgc/qTFmJ3YHp4FPTh+DlgCBQUw=")
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
` What can I do?